MODULE.bazel

```diff
@@ -167,7 +167,21 @@ register_toolchains("@llvm_18_toolchain//:all")
 # Go Toolchain
 # ====================================
 go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
-go_sdk.download(version = "1.23.0")
+go_sdk.download(version = "1.24.1")
```

Comment:
Does this change the version for both fips and non-fips?

Reply:
No the fips version is specified in the other "go_sdk" using the version attribute, they don't have to match